docs/deploy.md

```diff
@@ -82,3 +82,36 @@ For a full flag listing run the application with the `--help` parameter.
 ### environment variables
 
 cert-exporter respects the `NODE_NAME` environment variable.  If present it will add this value as label to file metrics.  See one of the [deployment yamls](./kops-nodes.yaml) for an example of using the [Kubernetes Downward API](https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/) to make use of this feature.
+
+### AWS
+
+cert-exporter is able to check secrets in AWS Secret Manager. The following arguments exist for monitoring certificates in AWS Secret Manager:
+
+```
+  -aws-account string
+        AWS account to search for secrets in
+  -aws-key-string string
```

Comment:
maybe `aws-key-substring`? since it's doing a substring search for this value?

Reply:
Sounds good to me! Applied it.